### PR TITLE
feat: 🎸 already implement this issue request

### PR DESCRIPTION
**Pull request by freddy50806**
sponsored talks go back in talk list!

✅ Closes: #865

(Thanks for sending a pull request! Please fill in the following content to let us know better about this change.)

## Types of changes
Please put an `x` in the box that applies

- [ * ] **Bugfix**
- [ ] **New feature**
- [ ] **Refactoring**
- [ ] **Breaking change** (any change that would cause existing functionality to not work as expected)
- [ ] **Documentation Update**
- [ ] **Other (please describe)**

## Description
**Describe what the change is**
let show sponsored lists in  talk list

## Steps to Test This Pull Request
The sponsored lists don't show in  talk list

## Expected behavior
We can see the sponsored list in talk list

## Related Issue
#865 

## More Information
image
image

### CODE REVIEW

File: src/templates/pycontw-2020/events/talk_list.html [hunk 1]
```diff
@@ -52,6 +52,17 @@ <h3 id="talk-category-{{ category }}">{{ category }}</h3>
...
 
+{% if sponsored_events %}
+<h3>{% trans 'Invited Events' %}</h3>
+<article>
+	<ul class="talk-list">
+		{% for event in sponsored_events %}
+		<li><p>{% blocktrans with event_title=event.title host_name=event.host.speaker_name sponsored_event_url=event.get_absolute_url %}<a class="proposal-title" href="{{ sponsored_event_url }}">{{ event_title }}</a> by {{ host_name }}{% endblocktrans %}</p></li>
+		{% endfor %}
+	</ul>
+</article>
```
[the remark this answers is not in the document]
Comment (freddy50806) [reply src/templates/pycontw-2020/events/talk_list.html:63]: 了解

[the remark this answers is not in the document]
Comment (freddy50806) [reply src/templates/pycontw-2020/events/talk_list.html]: solved at commit 22614a4

File: src/templates/pycontw-2020/events/talk_list.html [hunk 2]
```diff
@@ -52,6 +52,24 @@ <h3 id="talk-category-{{ category }}">{{ category }}</h3>
...
 
+{% if sponsored_events %}
+<article>
+	<h3>{% trans 'Invited Events' %}</h3>
+	<ul class="talk-list">
+		{% for event in sponsored_events %}
+		<li>
+			<p>
+				{% with event_title=event.title host_name=event.host.speaker_name sponsored_event_url=event.get_absolute_url %}
+				<a class="proposal-title" href="{{ sponsored_event_url }}">{{ event_title }}</a>
```
Comment (yychen) [inline src/templates/pycontw-2020/events/talk_list.html:63]:
```suggestion
				<a class="talk-title" href="{{ sponsored_event_url }}">{{ event_title }}</a>
```

**Review by yychen — APPROVED** [review]
Currently, the only sponsored event is also in community track, but we don't have the "label" field for sponsored event. I'll merge first, but maybe we shall specify it as community track.

**Pull request merged**